src/url.js

```diff
@@ -20,6 +20,7 @@ var mapValues = require('lodash/mapValues');
 var isString = require('lodash/isString');
 var isPlainObject = require('lodash/isPlainObject');
 var isArray = require('lodash/isArray');
+var isEmpty = require('lodash/isEmpty');
```

Comment:
lodash/isEmpty is big compared to other ones, or did you import it in other places already?